src/index.js

```diff
@@ -29,11 +31,32 @@ class ScrollableList extends Component {
     }
   }
 
+  refreshEvent() {
+    this.setState({ refreshing: true });
+
+    if (typeof this.onRefresh() === 'object' &&
+      typeof this.onRefresh().then === 'function') {
+      this.onRefresh().then(this.setState({ refreshing: false }));
+    } else {
+      this.onRefresh();
+      this.setState({ refreshing: false });
+    }
+  }
+
+  renderRefreshControl() {
+    return (
+      <RefreshControl
+        refreshing={this.state.refreshing}
+        onRefresh={this.refreshEvent.bind(this)} // eslint-disable-line react/jsx-no-bind
```

Comment:
This is cheating 😅 The binding should be done on the constructor, like suggested [here(https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#autobinding)!  Everything else looks 👌